fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/api/UniquenessViolationException.java

```diff
@@ -9,16 +9,16 @@
 /**
  * Translation of a duplicate key or value constraint SQLException.
  */
-public class DuplicateValueException extends DataAccessException {
+public class UniquenessViolationException extends DataAccessException {
```

Comment:
UniqueConstraintViolationException?

Reply:
I actual bounced between that and this one.  Will change to that.

Reply:
done